cypress/support/pages/learning_store/pageLearningStore.js

```diff
@@ -2,12 +2,21 @@
 
 class Learning_Store {
 
-  static #lernStoreOverviewNavigationButton = '[data-testid="Lern-Store"]'
+  static #learnStoreOverviewNavigationButton = '[data-testid="Lern-Store"]'
+  static #learnstoreMenuLink = '[data-testid="Lern-Store"]'
```

Comment:
Make the naming convention consistent. Either its `lernstore` or `lernStore`.

Reply:
changed all to learningstore.
The data-testid is not correct in my eyes, it should be in english.
But to change that a change in the application would be needed, maybe with support of devs. Do we really want to cause this effort?